Docs: Add initial CONTRIBUTING.md

carlexandre opened the pull request:
This is the **first version** of the contribution document, which helps maintainers understand how the contribution process should be carried out.

carlexandre committed:
Docs: Add initial CONTRIBUTING.md

Dev-Sams1012 commented:
Oi @carlexandre! Agradeço bastante fazer o _Pull Request_ da _Issue_ #2, com o arquivo `CONTRIBUTING.md`.

No entanto, há algumas partes do documento que acredito que possam ser deletadas, no caso, as citações/marcações dos nossos colegas que discutiram nos comentários da _Issue_.

Quando inclui esses pontos quando comecei o "rascunho" do arquivo estava considerando que seria apenas para nos guiar para criar o documento, a fim de mostrar que cada participante realmente colaborou na construção do `CONTRIBUTING.md`.

Você concorda com essa alteração? Acredito que fará mais sentido para os futuros colaboradores do projeto sem tais citações.
Se sim, me proponho a realizar tal alteração, editando o arquivo que você submeteu.

carlexandre commented:
Opa @Dev-Sams1012, sim!! Achei que fosse como citações que guiassem o leitor para entender as práticas de uma melhor forma, pode editar sim. Mas, deixo aqui um questionamento, você acha que seria interessante em alguns tópicos deixar esses textos, retirando as citações ou isso deixaria o documento muito extenso, prolixo e sem a objetividade que ele propõe?

Dev-Sams1012 commented:
> deixo aqui um questionamento, você acha que seria interessante em alguns tópicos deixar esses textos, retirando as citações ou isso deixaria o documento muito extenso, prolixo e sem a objetividade que ele propõe?

Bom, em alguns em específico deixarei, pois ajudam a criar a "atmosfera" do que a seção se refere.

Como em `"Após elencarmos algumas boas práticas isoladas para mantenedores no que diz respeito a contribuição, é importante que tenhamos noção do todo, isto é, de como essas boas práticas se inserem em um fluxo de contribuição."`

Acredito que fica interessante assim. E você? 

Com sua devida confirmação posso alterar os tópicos que mencionei anteriormente.

carlexandre commented:
Fica interessante sim. Pode fazer a alteração! Valeu Samuel

Dev-Sams1012 committed:
Update CONTRIBUTING.md

mthsfrts commented:
a ideia ficou legal, bem mais voltada ao projeto que a disciplina, acho que vcs poderiam so dar uma olhada nesse fork que eu tenho https://github.com/mthsfrts/waveterm.. ele eh um projeto open source que fui contruibuidor, la as diretrizes sao bem legais e bem detalhadas... acho que poderia ajudar mais a visao de vcs que ja ta bem legal, mas tem coisa que ta muito geral, poderia ser mais especifica

carlexandre committed:
Mudanças no CONTRIBUTING.MD.

Atualizei o arquivo para ser mais preciso e relacionado ao projeto. Tirando a maior parte dos textos que são mais educativos e gerais.

carlexandre commented:
Boa tarde, fiz algumas mudanças no CONTRIBUTING, de acordo com o feedback do @mthsfrts, tentanto especificar mais certos pontos e colocando links que facilitem o mantenedor a acessar novas issues, PR's e alguns guias.

O que vocês acharam?
